ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+# 2022-10-16
+
+ - Added a new `flake.flakePartsModules` option so a flake can expose a module
+   to be used in a downstream flake's flake-parts usage.
```

Comment:
```suggestion
 - Added a new `flake.flakePartsModules` option so a flake can expose a module
   to be used in a downstream flake's flake-parts usage. This will replace
   `.flakeModule` in the future.
```